pages/profile/read/views/profile.jsx

```diff
@@ -44,6 +44,18 @@ class Profile extends React.Component {
       over18 &&
       (correctEstablishment || !pilIncomplete);
 
+    let type;
+
+    if (pil && pil.status === 'revoked') {
+      type = 'pilReapply';
+    } else if (pilIncomplete) {
+      type = 'pilView';
+    } else {
+      type = 'pilApply';
+    }
```

Comment:
If you set `type = 'pilApply';` initially then you could omit this last condition but /shrug